.github/workflows/push.yml

```diff
@@ -28,3 +28,12 @@ jobs:
 
     - name: Run tests
       run: make test
+
+    - name: Checks for new endpoints against AWS WAF rules
+      uses: cds-snc/notification-utils/actions/waffles@v1
+      with:
+        app-loc: '${{ github.workspace }}'
```

Comment:
Not sure that is the proper way to [use the context variable](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) within quotes...